services-camel/src/test/java/edu/unc/lib/dl/services/camel/enhancements/EnhancementRouterIT.java

```diff
@@ -137,11 +154,11 @@ public void init() throws Exception {
     }
 
     @Test
-    public void testFolderEnhancements() throws Exception {
+    public void nonBinaryWithSourceImages() throws Exception {
```

Comment:
This test should verify that thumbnail enhancements are being called. You could also check that the access copy is not called.